package.json

```diff
@@ -67,8 +67,8 @@
     "pretty-quick": "^3.1.0",
     "rimraf": "^3.0.2",
     "string-replace-async": "^2.0.0",
-    "stylelint": "^13.13.0",
-    "stylelint-config-standard": "^22.0.0",
+    "stylelint": "^14.0.0",
+    "stylelint-config-standard-scss": "^2.0.1",
```

Comment:
This is the recommended way to migrate: https://stylelint.io/migration-guide/to-14/#syntax-option-and-automatic-inferral-of-syntax